x-pack/plugins/ml/public/components/job_select_list/job_select_service.js

```diff
@@ -137,13 +145,25 @@ export function JobSelectServiceProvider($rootScope, globalState) {
         txt = wholeGroups[0];
         if (wholeGroups.length > 1 || groupLessJobs > 0) {
           const total = (wholeGroups.length - 1) + groupLessJobs;
-          txt += ` and ${total} other${(total > 1 ? 's' : '')}`;
+          txt = i18n('xpack.ml.jobSelect.wholeGroupDescription', {
+            defaultMessage: '{wholeGroup} and {total, plural, zero {# other} one {# other} other {# others}}',
```

Comment:
do we need zero value of plural in this case?

Reply:
I believe yes